documents-d-archeologie-francaise.csl

```diff
@@ -105,7 +106,12 @@
   </macro>
   <macro name="responsability">
     <choose>
-      <if variable="author editor translator collection-editor" match="any">
+      <if variable="version">
+        <group suffix=" (éd.)">
```

Comment:
... but I can indeed use labels ...
Huge corrections done but this one to be done...